test/extensions/filters/network/mongo_proxy/config_test.cc

```diff
@@ -145,7 +145,7 @@ TEST(MongoFilterConfigTest, InvalidFaultsDelayPercent) {
       fixed_delay: 1s
     )EOF";
 
-    handleInvalidConfiguration(yaml_string, R"(invalid value -1 for type TYPE_UINT32)");
+    handleInvalidConfiguration(yaml_string, "percentage");
```

Comment:
Looks like the old error messages were more informative. Is that problematic or are we just minimizing the regex?

Reply:
The error messages are generated internally by protobuf and designed to be human readable. They change from time to time, which already caused churn in such tests before. I have shortened them to ensure that the error message has the offending field present. I think the rest of the error message is not important for the unit test.